dev/breeze/pyproject.toml

```diff
@@ -54,6 +54,7 @@ dependencies = [
     "jinja2>=3.1.0",
     "jsonschema>=4.19.1",
     "packaging>=23.2",
+    "pipx>=1.4.1",
```

Comment:
BTW. This is also useful in case of auto-upgrade. This will install `pipx` also in breeze's virtualenv so if you do not have pipx installed at `global` path, breeze will anyhow find it in the venv and will self-upgrade.